src/quo2/foundations/colors.cljs

```diff
@@ -223,8 +223,7 @@
                60 "#BD1E56"}})
 
 ;;;; Networks
-
-(def networks
+(def ^:private networks
```

Comment:
we should have the maps private so the mechanism is the way to interact with colors 👍